app/views/api/howto.html.haml

```diff
@@ -4,7 +4,7 @@
 
 %p
   Planning application data is available programmatically as feeds which can
-  be used in most web mapping APIs and desktop GIS software like #{link_to "Yahoo Pipes", "http://pipes.yahoo.com/"}. Details of the API are listed below.
+  be used in most web mapping APIs and desktop GIS software like <a href="https://zapier.com/">Zapier</a>. Details of the API are listed below.
```

Comment:
This should use the [Rails link_to helper](http://api.rubyonrails.org/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to) like the original.